lib/cmd_tool_n2ip.go

```diff
@@ -0,0 +1,70 @@
+package lib
+
+import (
+	"fmt"
+	"github.com/spf13/pflag"
+)
+
+// CmdToolN2IPFlags are flags expected by CmdToolN2IP
+type CmdToolN2IPFlags struct {
+	Help bool
+	ipv6 bool
+}
+
+// Init initializes the common flags available to CmdToolN2IP with sensible
+func (f *CmdToolN2IPFlags) Init() {
+	_h := "see description in --help"
+	pflag.BoolVarP(
+		&f.Help,
+		"help", "h", false,
+		"show help.",
+	)
+	pflag.BoolVarP(
+		&f.ipv6,
+		"ipv6", "6", false,
+		_h,
+	)
+}
+
+// CmdToolN2IP converts a number to an IP address
+func CmdToolN2IP(f CmdToolN2IPFlags, args []string, printHelp func()) error {
+	if len(args) == 0 {
+		printHelp()
+		return nil
+	}
+
+	expression := args[0]
+	if IsInvalidInfix(expression) {
+		return ErrInvalidInput
+	}
+
+	// n2ip also accepts an expression which is why the following
+	// Steps are being done
+	// Convert to postfix
+	// If it is a single number and not an expression
+	// The tokenization and evaluation would have no effect on the number
```

Comment:
Let's reduce the words on this:

```suggestion
	// NOTE: n2ip also accepts an expression, hence the tokenization and evaluation.
```